src/astro/files/operators/files.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+
+from airflow.models import BaseOperator
+from airflow.utils.context import Context
+
+from astro.files.locations import create_file_location
+
+
+class _ListFileOperator(BaseOperator):
+    """List the file available at path and storage
+
+    :param task_id: The task id for uniquely identify a task in a DAG
+    :param path: A path pattern for which you want to get a list of file
+    :param conn_id: connection id for the services
+    """
+
```

Comment:
These two properties should probably be `template_fileds` - we may avoid this if we use `File`